packages/nameguard-react/src/components/Search/SearchEmptyState.tsx

```diff
@@ -36,11 +36,7 @@ export const SearchEmptyState = () => {
   const { openModal } = useSearchStore();
   const exampleNames = examples.map((n) => n.name);
 
-  const {
-    data,
-    isLoading,
-    error: hadLoadingError,
```

Comment:
Not sure if I understand this change. Can you please confirm that we will continue handling the case of a loading error correctly?

Reply:
![Uploading Captura de Tela 2024-05-14 às 17.11.30.png…]()